src/forms/inputs/select.js

```diff
@@ -85,10 +101,13 @@ function Select (props) {
     className, // eslint-disable-line no-unused-vars
     enablePlaceholderOption,
     options,
+    optionGroups,
     placeholder,
     ...rest
   } = omitLabelProps(props)
-  const optionObjects = serializeOptions(options)
+  const optionObjects = optionGroups.length
```

Comment:
This code is a little hard to read. It might be simpler to define two constants, `optionObjects` and `optionGroupObjects`, and write another helper `serializeOptionGroups()` to create the latter.